youtube/templates/comments_page.html

```diff
@@ -36,7 +41,7 @@ <h2>Comments page {{ comments_info['page_number'] }}</h2>
 
         <div class="comments">
             {% for comment in comments_info['comments'] %}
-                {{ comments.render_comment(comment, comments_info['include_avatars']) }}
+                {{ comments.render_comment(comment, comments_info['include_avatars'], slim) }}
```

Comment:
Remove `slim` parameter from this line since the `render_comment` function signature wasn't changed.

Reply:
This was intentional: Since slim is only used on /watch, timestamp_links should be added.